readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -649,3 +707,61 @@ def _post_build(self):
             self.run(
                 "mv", temp_pdf_file, target_file, cwd=self.project_path, record=False
             )
+
+    # Introduced by Feature.ENABLE_MULTIPLE_PDFS
+    def _post_build_multiple(self):
+        """Internal post build to cleanup PDF output directory and leave only .pdf files."""
+
+        pdf_files = self._pdf_files_generated()
+        if not pdf_files:
+            raise PDFNotFound()
+
+        if not os.path.exists(self.absolute_host_tmp_root):
+            os.makedirs(self.absolute_host_tmp_root)
+
+        # We cannot use '*' in commands sent to the host machine, the asterisk gets escaped.
+        # So we opt for iterating from outside the container
+        pdf_file_names = []
+        for fname in pdf_files:
+            shutil.move(
+                fname,
+                os.path.join(self.absolute_host_tmp_root, os.path.basename(fname)),
+            )
+            pdf_file_names.append(os.path.basename(fname))
```

Comment:
We could probably use `self.run(..., escape=False)` or similar if we want to avoid this.